vm-virtio/src/vsock/csm/connection.rs

```diff
@@ -0,0 +1,633 @@
+// Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
When we copy code from Firecracker or crosvm I like to see the commit has that it was copied from in the commit message.